eXist-db/exist
# Feature/flexible installer

Remove the step of copying the scripts from `bin` to `installer/scripts` in favor of keeping the scripts in the latter. This way, the installer always has the original copy of the scripts. This does not affect those building an installer from source, and allows people to build further, derivative installers that do not contain local filepaths.

To test:
- build an installer from source with `./build.sh installer`;
- install eXist-TEST1 in a separate location;
- build an installer from eXist-TEST1;
- install eXist-TEST2 in a separate location; and
- ensure that eXist-TEST2 starts up with `./bin/startup.sh`.
